testplan/testing/multitest/base.py

```diff
@@ -773,6 +773,23 @@ def _run_serial_testcases(self, testsuite, testcases):
                     )
                     break
 
+        if parametrization_reports:
+            for param_report in parametrization_reports.values():
+                if param_report.entries:
+                    start_times = []
+                    end_times = []
+                    for testcase in param_report.entries:
+                        timer = getattr(testcase, "timer")
+                        start_time = getattr(timer["run"], "start")
```

Comment:
here you can use timer["run"].start and timer["run"]. end as they are just Interval types which is essentially a named tuple